frontend/src/helpers/validation/index.js

```diff
@@ -0,0 +1,4 @@
+export const validateEmail = (value) =>
+  value.match(
+    /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+  );
```

Comment:
Do we need to put this in a separate file?

Reply:
I think no because is too long row and in refactoring this function in other file with logic